src/mlpack/methods/ann/layer/layer_types.hpp

```diff
@@ -260,6 +261,7 @@ using LayerTypes = boost::variant<
     BaseLayer<SoftplusFunction, arma::mat, arma::mat>*,
     BaseLayer<RectifierFunction, arma::mat, arma::mat>*,
     BatchNorm<arma::mat, arma::mat>*,
+    InstanceNorm<arma::mat, arma::mat>*,
```

Comment:
If you add the layer to `MoreTypes` it should fix the build issue.

Reply:
Sorry for the confusion, I meant can you move `InstanceNorm`  to `MoreTypes`, so if you remove it from `LayerTypes` it should work.

Reply:
Thanks for the clarification.